### PR TITLE
Introducing master script. Bug fixes. Cleanup

### PR DESCRIPTION
**Pull request by tahir24434**
src/main/python/gcloud > python master_mm_hydra_setup.py -i <deployment_id/tag> -f setup_config.ini
- Parallel deployments now work for real.
- Adding retries in checking the hosts status.
- Introducing continue feature.
- Bug fixes

Signed-off-by: tahir24434 tahir.rauf1@gmail.com

### CODE REVIEW

**Comment by sushilks** [comment]
Please change director from gcloud to hydra-deploy

File: src/main/python/gcloud/setup_helpers.py [hunk 1]
```diff
@@ -214,6 +219,18 @@ def create_zk_conf_script(conf):
...
+
+def write_to_file(fname, str_to_write):
+    tfile = open(fname, 'w')
+    tfile.write(str_to_write)
+    tfile.close()
+
+
 def create_slave_conf_script(ip):
     (fd, pathname) = mkstemp(prefix="slave_conf_")
     tfile = os.fdopen(fd, "w")
```
Comment (sushilks) [inline src/main/python/gcloud/setup_helpers.py:236]: Please dont commit pyc files 
setup_helpers.pyc

File: src/main/scripts/master.py [hunk 2]
```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/python
-
-print ("I am alive                                                                                                          ,,")
```
Comment (sushilks) [inline src/main/scripts/master.py]: what's this for ?

File: src/main/python/gcloud/setup_helpers.py (hunk 1 above)
Comment (tahir24434) [inline src/main/python/gcloud/setup_helpers.py:236]: I am removing this file (Not adding).

Comment (sushilks) [inline src/main/python/gcloud/setup_helpers.py:236]: ok

**Pull request merged**